agents/openstack/fence_openstack.py

```diff
@@ -1,4 +1,4 @@
-#!@PYTHON@ -tt
+#!/usr/libexec/platform-python -tt
```

Comment:
Dont change these, as they're replaced to the path found by `./configure` during `make`.